tools/pyclient/src/molgenis_emx2_pyclient/client.py

```diff
@@ -499,16 +443,27 @@ def delete_schema(self, name: str = None):
         :returns: a success or error message
         :rtype: string
         """
+        if name is None:
```

Comment:
This check is useful in multiple places. Maybe it could be integrated into a new method?